prime-angular-frontend/src/app/shared/components/dialogs/content/error-logger/error-logger.component.html

```diff
@@ -0,0 +1,15 @@
+<mat-dialog-content>
+  <p>URL: {{ this.router.url }}</p>
+  <p>Environment: {{ this.config.environmentName }}</p>
+  <p>Version: {{ this.environment.version }}</p>
+  <mat-expansion-panel *ngIf="this.error.stack">
```

Comment:
Don't use `this` in templates.  The `public` variables are already scoped to the template.

Reply:
Also, the user will be confused by the stack trace.  I would probably only display outside of production.